src/utilities/device_utils.h

```diff
@@ -115,32 +230,13 @@
                    __FILE__, __LINE__);                                                      \
       assert(0); exit(1);                                                                    \
    }
+#endif
 
-#define HYPRE_ONEDPL_CALL(func_name, ...)                                                    \
-  func_name(oneapi::dpl::execution::make_device_policy(                                      \
-           *hypre_HandleComputeStream(hypre_handle())), __VA_ARGS__);
-
-#define HYPRE_SYCL_LAUNCH(kernel_name, gridsize, blocksize, ...)                             \
-{                                                                                            \
-   if ( gridsize[0] == 0 || blocksize[0] == 0 )                                              \
-   {                                                                                         \
-     hypre_printf("Error %s %d: Invalid SYCL 1D launch parameters grid/block (%d) (%d)\n",   \
-                  __FILE__, __LINE__,                                                        \
-                  gridsize[0], blocksize[0]);                                                \
-     assert(0); exit(1);                                                                     \
-   }                                                                                         \
-   else                                                                                      \
-   {                                                                                         \
-      hypre_HandleComputeStream(hypre_handle())->submit([&] (sycl::handler& cgh) {           \
-         cgh.parallel_for(sycl::nd_range<1>(gridsize*blocksize, blocksize),                  \
-            [=] (sycl::nd_item<1> item) { (kernel_name)(item, __VA_ARGS__);                  \
-         });                                                                                 \
-      }).wait_and_throw();                                                                   \
-   }                                                                                         \
-}
-
-#endif // defined(HYPRE_USING_CUDA)
+/* - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+ *      macros for wrapping vendor library calls for error reporting
+ * - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - */
 
+/* WM: Q - any need to macro guard these definitions? */
```

Comment:
like putting in HYPRE_USING_CUBLAS?

Reply:
Sure. I mainly ask for the sake of consistency. Other definitions (e.g. HYPRE_THRUST_CALL below) listed here are behind macro guards, so I just wanted to see whether we have a policy for this kind of thing.